Update Notes/2.1.1.md

```diff
@@ -18,3 +18,9 @@
  - Add exponent and percentage to calculator - u9g
  - Add total trophy fish count to /pv - Vixid
  - Allow hiding messages below a set skyblock level - nopo
+ - Add crimson isle quests to todo overlay - Vixid
+ - Add replace chat social options to both party and guild chat - Vixid
+ - Add crimson isle page to /pv - Vixid
```

Comment:
I dont think this is contained in this pr ```suggestion
```

Reply:
What do you mean? I just added all my prs that had the 2.1.1 milestone to it. Unless you want me to only add this pr?

Reply:
Each PR should add its own changes only to the update logs. Since your past PRs didnt do that (i think) its fine for you to add those, but since the crimson isle page pr is still open you should just add that change to that PR

Reply:
Ok, I'll add it in that one, walker suggested something to change in that pr, so I'll be doing another commit anyway.